packages/components/src/components/Alerts/overlay.tsx

```diff
@@ -21,19 +37,25 @@ function getOverlayAlertsMessage(alerts: SDK.ServerAPI.InferResponseType<SDK.Ser
     }
   });
 
-  const suffixText = warns !== 0 && errors !== 0 ? 'problems' : warns === 0 ? 'errors' : 'warns';
+  const suffixText =
+    warns !== 0 && errors !== 0 ? 'problems' : warns === 0 ? 'errors' : 'warns';
 
   const fontSize = 16;
 
   return {
     title: `${alerts.length} compiled ${suffixText}`,
     detail: (
       <Typography.Text strong style={{ color: '#fff', marginBottom: 0 }}>
-        <Typography.Text style={{ color: 'inherit', fontSize }}>Compiled with </Typography.Text>
+        <Typography.Text style={{ color: 'inherit', fontSize }}>
+          Compiled with{' '}
+        </Typography.Text>
         <Typography.Text strong style={{ color: Color.Red, fontSize }}>
           {errors} errors
         </Typography.Text>
-        <Typography.Text style={{ color: 'inherit', fontSize }}> and </Typography.Text>
+        <Typography.Text style={{ color: 'inherit', fontSize }}>
+          {' '}
```

Comment:
Is this empty string necessary?

Reply:
It is modified by lint, when you commit this file, it will be auto changed, so it looks like necessarily